### PR TITLE
Abderahim abbas homework

### PR DESCRIPTION
**Pull request by Aabderahim**
This PR for **0-arrays-loops-exercise**

### CODE REVIEW

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (maxbye3) [inline abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/.gitignore:1]: Only one .gitignore required (in the root) consider deleting this

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/0-arrays-loops-exercise.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
```
Comment (maxbye3) [inline abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/0-arrays-loops-exercise.iml:2]: Not sure about all these .iml, .xml files being committed/pushed!

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/app.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+/**
+ * BONUS:
+ * Question 8
+ * Create a variable called q8. Use the reduce method to set the value of q8 to
+ * the total of all the values stored in the numbers array.
+ * (Hint: see https://mdn.io/reduce)
+ */
+
+//let sum = 0
+let sum = a1.reduce((pNbre, cNbre) => pNbre + cNbre)
```
Comment (maxbye3) [inline abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/app.js:125]: Nice use of functional loops!
If your not going to reassign the sum value - consider using const

**Review by maxbye3 — COMMENTED** [review]
A lot of files being added that isn't related to homework.
Please consider removing!

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/.gitignore (hunk 1 above)
Comment (Aabderahim) [reply abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/.gitignore:1]: sure, these files are created automatically under cache .idea. To fix this issue, i proposed on my branch to modify principal .gitignore.

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/0-arrays-loops-exercise.iml (hunk 2 above)
Comment (Aabderahim) [reply abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/.idea/0-arrays-loops-exercise.iml:2]: Same issue than .idea cache directory.

File: abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/app.js (hunk 3 above)
Comment (Aabderahim) [reply abderahim-abbas/homework/arrays-loops/0-arrays-loops-exercise/app.js]: Fixed.

**Pull request merged**